routes/users.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const { signup, login } = require('../controllers/user');
+const route = express.Router();
```

Comment:
Alway give an extra line between imported model and module in the file
```suggestion
const { signup, login } = require('../controllers/user');

const route = express.Router();
```

Reply:
I have noted this and made the correction